server/middleware/__tests__/configureEstablishment.spec.js

```diff
@@ -0,0 +1,41 @@
+const { configureEstablishment } = require('../configureEstablishment');
+
+describe('configureEstablishment', () => {
+  const next = jest.fn();
+
+  beforeEach(() => {
+    next.mockClear();
+  });
+
+  it('should use the set the session from the config', () => {
+    const req = {
+      session: {
+        establishmentName: 'wayland',
+        establishmentId: 793,
+      },
```

Comment:
I _think _ this test might be flawed 🤔 The establishment name from the header will return `wayland` and generate the id from that. The name and id being passed through are coming from that default and _not_ the session. I think. 🤯

Reply:
there's no establishment in the header here? 

I think you're right though it is wrong - the existing test just checked that it used the default prison:
`req.headers?.host || 'wayland'`

I'll split into 2 separate tests:
* one that uses the default 
* one that uses the session

(think we should ditch the defaulting behaviour everywhere though as it most likely going to silently show the wrong thing)

This was only existing test (the file didn't have `.spec.` in it so I've fixed it.

Reply:
Yeah... I didn't expand the collapsed if statement 😬 In the right ballpark though...